products/grafana.md

```diff
@@ -18,17 +18,24 @@ auto:
 # - support(x) = releaseDate(x+1)
 # - eol(x) = releaseDate(x+2)
 releases:
+-   releaseCycle: "10.1"
+    releaseDate: 2023-08-23
+    support: true
+    eol: false
+    latest: "10.1.0"
+    latestReleaseDate: 2023-08-23
+
 -   releaseCycle: "10.0"
     releaseDate: 2023-06-09
-    support: true
+    support: 2023-08-23
     eol: false
     latest: "10.0.4"
     latestReleaseDate: 2023-08-23
 
 -   releaseCycle: "9.5"
     releaseDate: 2023-04-06
     support: 2023-06-09
-    eol: false
+    eol: 2023-08-23
```

Comment:
9.5 should still be maintained according to release comments:

> After 9.0, 2 latest minors are supported, **along with the last minor of the previous major**.

Reply:
You are right, I changed back to `false` and added a comment.